contracts/token/erc20/README.md

```diff
@@ -3,11 +3,14 @@
 This directory contains ERC 20 token contracts that game studios could choose to use
 directly or extend. 
 
-| Contract                        | Description                                   |
-|---------------------------------|-----------------------------------------------|
-| ImmutableERC20                  | Provides basic ERC 20 capability. Designed to be extended. | 
+| Contract                               | Description                                   |
+|--------------------------------------- |-----------------------------------------------|
+| preset/ImmutableERC20MinterBurnerPermit| Provides basic ERC 20 Permit, Capped token supply and burn capability. Designed to be extended.  | 
```

Comment:
Is this still correct, "Designed to be extended" ?  Do we envisage game studios will deploy this as is?